instrumentation/net/http/otelhttp/client_test.go

```diff
@@ -0,0 +1,73 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package otelhttp
+
+import (
+	"context"
+	"net/http"
+	"net/http/httptest"
+	"net/url"
+	"strings"
+	"testing"
+
+	"go.opentelemetry.io/otel/api/global"
+
+	mocktrace "go.opentelemetry.io/contrib/internal/trace"
+)
+
+func TestConvenienceWrappers(t *testing.T) {
+	provider, tracer := mocktrace.NewTracerProviderAndTracer(instrumentationName)
+	oldProvider := global.TracerProvider()
+	global.SetTracerProvider(provider)
+	defer global.SetTracerProvider(oldProvider)
```

Comment:
This will have no effect, the TracerProvider can only be set once.

https://github.com/open-telemetry/opentelemetry-go/blob/8ed55f5980611f2e1effe471f9946aba263f255d/api/global/internal/state.go#L54-L68

Setting the `DefaultClient` here instead of relying on the global package would ensure the following tests use the intended Tracer and TextMapPropagator.

Reply:
Cool, removed the attempt to re-set the global provider.